gst/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.c

```diff
@@ -79,7 +171,18 @@ tflite_loadModelFile (const GstTensorFilter * filter, void **private_data)
   }
   tf = g_new0 (tflite_data, 1); /** initialize tf Fill Zero! */
   *private_data = tf;
-  tf->tflite_private_data = tflite_core_new (filter->prop.model_file);
+
+  int use_nnapi = tflite_get_parser_boolean (filter, TFLITE_OPTION_NNAPI);
```

Comment:
This function tries to parse the whole option string, and yet, tries to parse boolean values only for a single property.

With a single function call, it should configure the object, ```tf```, for all possible argument.